packages/benchmarks/lit-html/stub1/stub1.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<script type="module">
+  import { html, render } from "lit-html";
+
+  (async () => {
+    performance.mark("start");
+    for (let i = 0; i < 1000; i++) {
+      render(html`<p>Hello World</p>`, document.body);
+    }
+    // Wait for LitElement render microtasks to complete.
+    await Promise.resolve(null);
```

Comment:
Hmm, I think you meant to put this in the lit-element test? (`render` is sync in lit-html).  Not a huge deal since these are stubs.

Reply:
Oops, yeah I updated the wrong file when adding that await. Fixed.